pkgs/development/python-modules/rdkit/default.nix

```diff
@@ -19,28 +19,29 @@
 }:
 let
   external = {
-    avalon = fetchzip {
-      url = "http://sourceforge.net/projects/avalontoolkit/files/AvalonToolkit_1.2/AvalonToolkit_1.2.0.source.tar";
-      sha256 = "0nhxfxckb5a5qs0g148f55yarhncqjgjzcvdskkv9rxi2nrs7160";
-      stripRoot = false;
+    avalon = fetchFromGitHub {
+      owner = "rohdebe1";
+      repo = "ava-formake";
+      rev = "AvalonToolkit_2.0.2";
+      hash = "sha256-YI39OknHiSyArNGqRKrSVzEJnFc1xJ0W3UcTZrTKeME=";
```

Comment:
Upstream changed to fetch from github: https://github.com/rdkit/rdkit/blob/Release_2022_09_1/External/AvalonTools/CMakeLists.txt#L25